JavaScript/JavaScriptSDK/Sender.ts

```diff
@@ -368,12 +401,20 @@ module Microsoft.ApplicationInsights {
         public _xhrReadyStateChange(xhr: XMLHttpRequest, payload: string[], countOfItemsInPayload: number) {
             if (xhr.readyState === 4) {
                 if ((xhr.status < 200 || xhr.status >= 300) && xhr.status !== 0) {
-                    this._onError(payload, xhr.responseText || xhr.response || "");
+                    if (!this._config.disableRetry() && this._isRetriable(xhr.status)) {
```

Comment:
Yeah, we can't rename existing switches (somebody can be using them in a snippet). `disableRetry` is new, I'll rename it to `isRetryDisabled`. Thanks!